src/AasxServerStandardBib/Database.cs

```diff
@@ -0,0 +1,258 @@
+﻿using MongoDB.Bson.Serialization.Serializers;
+using MongoDB.Bson.Serialization;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AasxServerStandardBib.Exceptions;
+using MongoDB.Bson;
+using MongoDB.Driver.Linq;
+using System.Collections;
+using AasCore.Aas3_0;
+using AdminShellNS;
+using MongoDB.Driver.GridFS;
+using static AasxServerStandardBib.TimeSeriesPlotting.PlotArguments;
+using System.IO;
+
+
+//Author: Jonas Graubner
+//contact: jogithub@graubner-bayern.de
+namespace AasxServerStandardBib
+{
+    public interface IDatabase
```

Comment:
Please avoid creating multiple classes/interfaces in a single file. This is considered a Code Smell. Although this exists in other parts of the project, let's not add new instances.

More info: [Code Smell #228: Multiple Classes per File](https://maximilianocontieri.com/code-smell-228-multiple-classes-per-file)